### ISSUE TITLE
Typo in documentation page /reference/storage.html

### ISSUE THREAD

**Issue by lhqing** [skypilot-org/skypilot#1869]
Hi, Skypilot team

I think there is a typo in "With mount mode, files are copied at VM initialization and any writes to the mount path will not be replicated on the object store."
You probably mean "With COPY mode"

```
sky.Storage.mode: str; either of MOUNT or COPY, defaults to MOUNT
    Whether to mount the storage object by copying files, or actually
    mounting the remote storage object. With MOUNT mode, files are streamed
    from the remote object store and writes are replicated to the object
    store (and consequently, to other workers mounting the same Storage).
    With mount mode, files are copied at VM initialization and any writes to
    the mount path will not be replicated on the object store.
```

**Comment by Michaelvll** [comment]
Great catch! Yes, it should be copy mode for the last sentence. Thanks for raising the issue @lhqing! Would you like to submit a PR for that? : )

**Comment by lhqing** [comment]
Sure, just modify the rst file OK? #1870

**Comment by Michaelvll** [comment]
Fixed by #1870. Closing. Thanks for fixing it @lhqing!
